reactive/magpie.py

```diff
@@ -33,9 +33,10 @@ def check_check_state(magpie):
     '''
     Servers should only update their status after iperf has checked them
     '''
-    if (magpie.get_iperf_checked() is not None) and \
-            (hookenv.local_unit() in magpie.get_iperf_checked()):
-        set_state('iperf.checked')
+    if (magpie.get_iperf_checked() is not None):
```

Comment:
Does this have to be a None check rather than just "if magpie.get_iperf_checked():"